perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java

```diff
@@ -114,6 +114,15 @@
 	 */
 	List<Facility> getFacilities(PerunSession perunSession) throws PrivilegeException;
 
+	/**
+	 * Gets all enriched facilities.
```

Comment:
It should be more like "Gets all enriched facilities user has access rights to." Similar to getFacilities javadoc.

Reply:
Fixed.